.claude/S&P.md

```
@@ -23,3 +23,1913 @@ Review this file before making changes to the codebase.
+<summary>📝 Suggested patch pattern</summary>
+
+```diff
+-```
++```text
+ Verify each finding against the current code and only fix it if needed.
+ ```
+
+-````
++````text
+ Verify each finding against the current code and only fix it if needed.
+ ````
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)